### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MarcusTXK**

### CODE REVIEW

**Review by tyanhan — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Everything being offline helps ensure that all information is kept private and w
 
 To setup MYTranslator on your local device, follow thesse steps:
```
Comment (tau-bar) [inline README.md]: change to Hearsay

**Review by tau-bar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MarcusTXK) [reply README.md]: Ok

**Review by tyanhan — APPROVED** [review]
LGTM

**Review by tau-bar — APPROVED** [review]
lgmt

**Comment by tau-bar** [comment]
> LGTM

u didn't find the mistake shame on u

**Pull request merged**